src/main/java/net/syscon/elite/service/CourtHearingsService.java

```diff
@@ -130,7 +130,7 @@ public CourtHearing scheduleHearing(final Long bookingId, final PrisonToCourtHea
     /**
      * Returns all court hearings for a given booking ID for the given date range.
      */
-    @VerifyBookingAccess
+    @VerifyBookingAccess(overrideRoles = "COURT_HEARING_MAINTAINER")
     public CourtHearings getCourtHearingsFor(final Long bookingId, final LocalDate fromDate, final LocalDate toDate) {
```

Comment:
is this missing a PreAuthorize?

Reply:
Do we need to restrict for reads?  Think this doesn't need the role, Ignore me, this is to allow unfettered access to court hearings if you have the role right?, maybe add the SYSTEM_USER role in to?

Reply:
Done.